packages/web3-react/package.json

```diff
@@ -43,7 +43,7 @@
     "@web3-react/walletlink-connector": "6.2.5",
     "tiny-invariant": "^1.1.0",
     "tiny-warning": "^1.0.3",
-    "ua-parser-js": "^0.7.28",
+    "ua-parser-js": "1.0.2",
```

Comment:
```suggestion
    "ua-parser-js": "^1.0.2",
```

Reply:
@Jabher I decided to use this strict 1.0.2 version intentionally. We also use it at connect-wallet-modal. I like to have more explicit control over what exact version is installed, considering that `ua-parser-js` has rare releases.